From 7ea699cff27f60b731ce1c324304faa998c6191a Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Tue, 11 Apr 2017 15:08:19 +0800
Subject: [PATCH 1657/5242] MLK-14671 input: touch: focaltech: disable debug
 information

commit  d04dd17d736bc9f65214935e98de9f75daeefe2f from
https://source.codeaurora.org/external/imx/linux-imx.git

disable debug information for focaltech touch.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry picked from commit b88a0ed7837ab964253f710da03fdab481592350)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../touchscreen/focaltech_touch/focaltech_config.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/focaltech_touch/focaltech_config.h b/drivers/input/touchscreen/focaltech_touch/focaltech_config.h
index 51de298..1d19f53 100644
--- a/drivers/input/touchscreen/focaltech_touch/focaltech_config.h
+++ b/drivers/input/touchscreen/focaltech_touch/focaltech_config.h
@@ -92,7 +92,7 @@
  * show debug log info
  * enable it for debug, disable it for release
  */
-#define FTS_DEBUG_EN                            1
+#define FTS_DEBUG_EN                            0
 
 /*
  * Linux MultiTouch Protocol
-- 
1.7.9.5

